tests/unit/models/package-info-cache-test.js

```diff
@@ -0,0 +1,219 @@
+'use strict';
+
+const path = require('path');
+const expect = require('chai').expect;
+const Project = require('../../../lib/models/project');
+let addonFixturePath = path.resolve(__dirname, '../../fixtures/addon');
+const MockUI = require('console-ui/mock');
+const MockCLI = require('../../helpers/mock-cli');
+
+describe('models/package-info-cache.js', function() {
+  let project, projectPath, packageJsonPath, packageContents, projectPackageInfo, ui, cli, pic;
+  this.timeout(100000);
+
+  beforeEach(function() {
+    ui = new MockUI();
+    cli = new MockCLI({ ui });
+  });
+
+  describe('packageInfo contents tests on valid project', function() {
+    beforeEach(function() {
+      projectPath = path.resolve(addonFixturePath, 'simple');
+      packageJsonPath = path.join(projectPath, 'package.json');
+      packageContents = require(packageJsonPath);
+      project = new Project(projectPath, packageContents, ui, cli);
```

Comment:
The PackageInfoCache is generally used by creating it inside a Project or Addon, and then calling 'loadProject' or 'loadAddon' on that object. That's how it starts the discovery process. The PackageInfo and other internal objects there are really just supposed to be created in the PIC itself. Make sense?

Reply:
yeah! since it's `package-info` unit test, my assumption was that we would bypass `Project` or `Addon` altogether and will be testing `PackageInfoCache` directly. Looking at the current implementation, we pass `this` to both `loadProject` and `loadAddon` so my first thought was to avoid passing `this` and pass _only_ the arguments we need to minimize api surface and things the method relies on

Reply:
Well, we could create a new PackageInfoCache(), then call loadProject on it to load a project, since that's the public API, but creating the project itself already creates a PackageInfoCache as part of its constructor, so it's sort of pointless. We don't really want to call _readPackageInfo directly, since it's internal.